Flaky test in test_inference_tiles.py

## Description and motivation

There is a test that seems to be [failing occasionally](https://travis-ci.com/github/IBM/aihwkit/jobs/485511907#L1113): 

```
tests/test_inference_tiles.py:160: in test_post_forward_modifier_types
    self.assertNotAlmostEqualTensor(x_output, x_output_post)
    tests/helpers/testcases.py:36: in assertNotAlmostEqualTensor
    assert_raises(AssertionError, self.assertTensorAlmostEqual, tensor_a, tensor_b)
E   AssertionError: AssertionError not raised by assertTensorAlmostEqual
```

## Proposed solution

Check the test and update it as needed - it seems an issue with the test rather than with the code.

## Alternatives and other information

<!-- A clear and concise description of any alternative solutions or features you've considered. -->

--- a/src/aihwkit/simulator/configs/devices.py
+++ b/src/aihwkit/simulator/configs/devices.py
@@ -533,8 +533,8 @@ class SoftBoundsPmaxDevice(SoftBoundsDevice):
     where :math:`B=\frac{r_\text{max} -
     r_\text{min}}{1 - e^{-\alpha p_\text{max}}}`.
 
-    Here :math:`p_max` is the number of pulses that were applied to get the
-    device from the minimum conductance (minimum of range,
+    Here :math:`p_\text{max}` is the number of pulses that were applied to get
+    the device from the minimum conductance (minimum of range,
     :math:`r_\text{min}`) to the maximum (maximum of range,
     :math:`r_\text{max}`).
 

--- a/tests/test_inference_tiles.py
+++ b/tests/test_inference_tiles.py
@@ -127,7 +127,7 @@ class InferenceTileTest(ParametrizedTestCase):
         """Tests whether post update diffusion is performed."""
         rpu_config = self.get_rpu_config()
         rpu_config.drift_compensation = None
-        rpu_config.forward.w_noise = 0.0
+        rpu_config.forward.is_perfect = True
         rpu_config.forward.out_noise = 0.0
         rpu_config.forward.inp_noise = 0.0
 

